src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -509,7 +509,20 @@ export module DotNet {
     return value;
   });
 
-  class DotNetStream {
+  export interface IDotNetStreamReference {
```

Comment:
In Typescript it's idiomatic for interfaces not to have an `I` prefix. I suppose the difference vs C# is that it's very meaningful to think about runtime types in .NET but not so relevant in JS.